Pipfile

```diff
@@ -32,6 +32,7 @@ unidecode = "*"
 httpx = ">=0.12.0"
 geopy = "*"
 py-mini-racer = "*"
+geojson-pydantic = "*"
```

Comment:
I allowed myself to add this tiny dependency, the only interest is to include some documentation about what the geojson schema looks like to potential user who would not be used to it.

I'm not sure if this is a good idea though, but it is quite tiny (just contains some pydantic models) so I don't think we have to be afraid of it not being well maintained or anything on this range. I'd willingly remove it if you think it's a bad idea, the provided example can be quite enough ^^